src/main/java/innovimax/mixthem/operation/ResultType.java

```diff
@@ -0,0 +1,22 @@
+package innovimax.mixthem.operation;
+
+/**
+* <p>This is a enumeration of result type.</p>
+* <p>Here are the types:</p>
+* <ul>
+* <li>_GET_RESULT: get the result</li>
+* <li>_IGNORE_RESULT: ignore the result</li>
+* <li>_READ_FIRST_FILE: read first file</li>
+* <li>_READ_SECOND_FILE: read second file</li>
+* <li>_READ_BOTH_FILES: read both files</li>
+* </ul>
+* @author Innovimax
+* @version 1.0
+*/
+public enum ResultType {
+    _GET_RESULT,
```

Comment:
why do they start by "_" ?

Reply:
Same as in Rule, ParamType, AtltMode, ZipType...